futurenhs.app/pages/_app.page.tsx

```diff
@@ -15,6 +16,25 @@ const CustomApp = ({ Component, pageProps }) => {
     const { errors, layoutId } = pageProps;
 
     const hasServerError: boolean = errors?.filter(error => Object.keys(error).filter(key => Number(key) >= 500)).length > 0;
+    let hasFormErrors: boolean = false;
```

Comment:
I wonder if we can make this even simpler - The service manual only explicitly discusses adding "Error: " on form validation issues, but I'm not sure if there's any guidance about appending that for any page error. I can't personally see why you wouldn't do that. In that case you'd be able to add it with a simple check that props.errors has keys. We'll leave like this for now, but I'll ask Neil C what he thinks